### ISSUE TITLE
Add value check before duty execution

### ISSUE THREAD

**Issue by alonmuroch** [bloxapp/ssv#176]
The below validator got slashed because the nodes re-did the same duty twice. 
Since we currently do not have slashing protection this actually slashed the validator which would not have happened with slashing protection.

This brought to attention another issue which requires checking the value before starting an iBFT instance, if we won't do it a situation where nodes start an instance with slash able data will cause them to never decide as we have a value check precondition in the pre-prepare pipeline.

https://prater.beaconcha.in/block/801402#attester-slashings
